deployment/docker-compose.yml

```diff
@@ -10,6 +10,35 @@ services:
         volumes:
             - pgdata-example:/opt/iqgeo/data/pgdata
 
+    iqgeo-tools:
```

Comment:
Would a better name be `iqgeo-worker`? I know it wouldn't match the image name, but maybe makes it clearer what the service is about